document/FileFormat.mdk

```diff
@@ -2696,31 +2708,385 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | the branch is not           |                             |                             |
+|                  | known.                      |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `call`           | Recommended: The            | Optional: An array          | Optional: The names and     |
+|                  | fully qualified name of     | containing the values of    | values of any variables     |
+|                  | the function being          | the arguments               | in the expressions passed   |
+|                  | called.                     | passed to the               | as the arguments to the     |  
+|                  |                             | function. This array        | function. For object        |
+|                  |                             | shall not include the       | method calls, this may      |
+|                  |                             | implicit object reference   | include the name and value  |
+|                  |                             | (for example, `this`)       | of the object on which      |
+|                  |                             | passed to object            | the method was invoked.     |
+|                  |                             | method calls.               |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `callReturn`     | Recommended: The            | Optional: If the function   | Absent                      |
+|                  | fully qualified name of     | returns a value or          |                             |
+|                  | the  function being         | values, in which case,      |                             |
+|                  | returned from.              | the value or values         |                             |
+|                  |                             | returned.                   |                             |
+|                  |                             | Absent: If the function     |                             |
+|                  |                             | does not return a value.    |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `continuation`   | Absent                      | Absent                      | Optional: The names and     |
+|                  |                             |                             | values of selected          |
+|                  |                             |                             | variables at the specified  |
+|                  |                             |                             | location. Any variable whose|
```

Comment:
I'd rather not. I think "Any variable whose value is available" is a clear, language-independent way to express this.

---
In reply to: [77724880](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77724880) [](ancestors = 77724880)